### ISSUE TITLE
Add more information to exported proforma xml

### ISSUE THREAD

**Issue by Mathis-Z** [openHPI/codeharbor#941]
Currently there is some information about tasks and tests that does not get imported/exported to and from proforma.
This includes for example:

- [ ] the tasks license
- [ ] the tasks labels
- [ ] the testing framework for tests
- [ ] the private/public setting for tasks

Maybe these should be included in some form.

**Comment by kkoehn** [comment]
What would be the advantage of adding that information to an export?

Since that data is not part of the official ProFormA specification, no tool would use it except CodeHarbor itself. If we reimport the exported data, I'd expect the data in CodeHarbor to be at least up-to-date or even more recent than the exported data. It might make sense if you want to import the task as "new Task", but in that case you could just use the duplicate feature.

I don't think this is needed.

**Comment by MrSerth** [comment]
You're right about missing support in the ProFormA standard for everything except the testing framework. 

If I see this right, the [`unittest` subtype](https://github.com/ProFormA/proformaxml/blob/master/proforma-unittest.xsd) supports specifying a framework and version. This would be something we could add.
